nixos/tests/docker-tools.nix

```diff
@@ -378,5 +378,10 @@ import ./make-test-python.nix ({ pkgs, ... }: {
         docker.succeed(
             "docker run --rm ${examples.layeredImageWithFakeRootCommands.imageName} sh -c 'stat -c '%u' /home/jane | grep -E ^1000$'"
         )
+
+    with subtest("exportImage produces a valid tarball"):
+        docker.succeed(
+            "tar -tf ${examples.exportBash} > /dev/null"
+        )
```

Comment:
Could you check whether some file is present?
A change that omits or moves files should be caught.

Reply:
We can check for the existance of bash, since we're testing on the bash docker image it should exist in bin/bash. I've pushed a commit that checks this path is in the archive.